public/js/collection_management.js

```diff
@@ -1,5 +1,5 @@
 var getUrl = window.location;
-var baseUrl = getUrl.protocol + "//" + getUrl.host + "/" + getUrl.pathname.split('/')[1];
```

Comment:
Sometimes when hitting the url, it generates `localhost:8080/collectioncollection/add` .  This pr addresses that.

Reply:
I can sorta see how that would work. I think it might be best to eventually segregate the collection adding and the game adding js logic from each other.